frontend/vite.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
+})
```

Comment:
Going forward, once we add in the backend routes we can use a http-proxy here that pushes `/api/...` requests to the respective APIs.